{"name": "Ошибки авторизации с неверным логином/паролем", "status": "passed", "steps": [{"name": "Регистрация нового курьера и возвращение логина и пароля", "status": "passed", "start": 1734291879228, "stop": 1734291879394}, {"name": "Авторизация курьера", "status": "passed", "parameters": [{"name": "login", "value": "'test_login'"}, {"name": "password", "value": "'wrong_password'"}], "start": 1734291879394, "stop": 1734291879525}], "parameters": [{"name": "wrong_login", "value": "'test_login'"}, {"name": "wrong_password", "value": "'wrong_password'"}], "start": 1734291879228, "stop": 1734291879526, "uuid": "5a5ce5bf-00a5-406d-a6fd-30cec89934d2", "historyId": "0fb2c6eac62e7237b3ea31d754a1aeda", "testCaseId": "1f2bce4b16f18147086031658deae890", "fullName": "tests.test_login_courier.TestCourierAuthorization#test_authentication_errors", "labels": [{"name": "suite", "value": "Тесты на проверку логина курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestCourierAuthorization"}, {"name": "host", "value": "LAPTOP-PCJM6AA3"}, {"name": "thread", "value": "19776-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}